editor/editor_settings.cpp

```diff
@@ -724,6 +724,11 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	/* Run */
 
 	// Window placement
+#ifdef ANDROID_ENABLED
+	// Should match the ANDROID_WINDOW_* constants in 'platform/android/java/editor/src/main/java/org/godotengine/editor/GodotEditor.kt'
+	String android_window_hints = "Auto (based on screen size):0,Same as Editor:1,Side-by-side with Editor:2";
+	EDITOR_SETTING(Variant::INT, PROPERTY_HINT_ENUM, "run/window_placement/android_window", 0, android_window_hints)
+#else // ANDROID_ENABLED
```

Comment:
It cannot be in the docs if you make it conditional, as the docs are generated on Linux.

Reply:
Fixed!